chcemvediet/apps/inforequests/tests/test_models/test_inforequest.py

```diff
@@ -7,16 +7,20 @@
 from django.db import IntegrityError
 from django.contrib.auth.models import User
 from django.utils import timezone
+from django.utils.http import urlencode
+from django.utils.translation import ugettext_lazy as _
 from django.test import TestCase
+from django.utils.translation import ugettext as __
```

Comment:
Nie je dobre ak v jednom subore mame dva aliasy na gettext. Nie je sanca si v kode pamatat, ktory je `_` a ktory je `__`.

Potrebujes naozaj aj `ugettext_lazy` aj `ugettext`? V testoch zvycajne `ugettext_lazy` netreba, kedze to aj tak vyhodnocujes hned, nie "lazy". Ak nie, tak importni iba `ugettext`.

Ak predsa potrebujes oba, tak potom aslias daj iba jednemu (alebo ziadnemu) z nich.

Btw, tiez nizsie v kode `_` pouzivas ako placeholder pre unused premenne. Takze alias `_` pre gettext nie je dobry.

Reply:
Nie netreba nam `ugettext_lazy`. Nepouzil som ani alias. :ok: